demos/python_demos/face_recognition_demo/face_recognition_demo.py

```diff
@@ -122,12 +129,16 @@ def __init__(self, args):
 
         log.info("Loading models")
         face_detector_net = self.load_model(args.m_fd)
+        if args.fd_input_height and args.fd_input_width :
```

Comment:
A user should get a warning if only `fd_input_height` or `fd_input_width` was provided.